back/src/users/bulk-creation/validations.ts

```diff
@@ -54,11 +54,9 @@ export const companyValidationSchema = yup.object().shape({
     .ensure()
     .compact()
     .required()
+    .min(1)
     .test("is-companyType", "${value} is not a valid company type", value => {
-      const isCompanyType = value.reduce((acc: boolean, curr: string) => {
-        return acc && COMPANY_TYPES.includes(curr);
-      }, true);
-      return isCompanyType;
+      return value.every(type => COMPANY_TYPES.includes(type));
```

Comment:
Bien vu pour la simplification 👍 On pourrait aller plus loin et utiliser un `array().of(string().oneOf(COMPANY_TYPES))`